### PR TITLE
chore: release preparation

### PR DESCRIPTION
**Pull request by bhumitra**
### Summary

- Updated `PowerValidatedSolutions` from v13.0.0 to v13.1.0
- Updated `VMware.PowerCLI` from v2.5.0 to v2.6.0. 
- Updated module versions in `docs/snippets/install-module.ps1` 
- Added PowerShell Gallery downloads badge to the `docs/index.md` 

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [x] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

- [x] Tests have been completed.
- [ ] Documentation has been added or updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 
 ## [v2.4.0](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/releases/tag/v2.4.0)
 
 Bugfix:
 
-- Updated `Request-VcenterStorageHealth` to exclude `/dev/mapper/archive_vg-archive` from the output per [KB 76563](https://kb.vmware.com/s/article/76563).
+- Updated `Request-VcenterStorageHealth` to exclude `/dev/mapper/archive_vg-archive` from the output per [KB 76563](https://kb.vmware.com/s/article/76563). [GH-167](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/167)
+
+Chore:
+
```
Comment (tenthirtyam) [inline CHANGELOG.md:12]:
```suggestion

- Updated `VMware.PowerCLI` from v13.0.0 to v13.1.0. [GH-171](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/171)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+- Updated `PowerValidatedSolutions` from v2.5.0 to v2.6.0. [GH-171](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/171)
+- Updated module versions in `docs/snippets/install-module.ps1` [GH-171](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-reporting/pull/171)
```
Comment (tenthirtyam) [inline CHANGELOG.md:14]:
```suggestion
```

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Just a minor change.

**Comment by bhumitra** [comment]
> Just a minor change.

Addressed

**Review by tenthirtyam — APPROVED** [review]
LGTM!

**Pull request merged**